packages/mdc-chips/action/index.ts

```diff
@@ -23,7 +23,13 @@
 
 export * from './adapter';
 export * from './component';
-export * from './constants';
 export * from './foundation';
 export * from './primary-foundation';
 export * from './trailing-foundation';
+export * from './types';
+export {
+  CssClasses as ChipActionCssClasses,
```

Comment:
I'm not attached to this naming scheme, but I went with it because I saw something similar in the other entry points.